config/crd/bases/kubedl.io_marsjobs.yaml

```diff
@@ -32,81 +32,57 @@ spec:
     name: v1alpha1
     schema:
       openAPIV3Schema:
-        description: MarsJob represents a mars job instance nad
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
how did you make these changes, are these description deleted manually or by some commands

Reply:
re-run `make manifests` command after update `CRD_OPTIONS` in Makefile, the output manifests files will truncate these fileds.